### PR TITLE
Enabled Azure integration tests in Jenkins

### PR DESCRIPTION
**Pull request by JunqiZhang0**
Enabled jenknins Fix #1387

### CODE REVIEW

**Comment by samvarankashyap** [comment]
[test]

File: linchpin/provision/roles/azure/tasks/provision_azure_vm.yml [hunk 1]
```diff
@@ -79,7 +79,7 @@
     image: "{{ image | default(omit) }}"
   register: res_def_output
   when: not _async
-  no_log: "{{ not debug_mode }}"
+  # no_log: "{{ not debug_mode }}"
```
Comment (14rcole) [inline linchpin/provision/roles/azure/tasks/provision_azure_vm.yml]: I assume you don't want to commit this?

Comment (JunqiZhang0) [reply linchpin/provision/roles/azure/tasks/provision_azure_vm.yml]: Yes, I fixed it, I commented it out only for debugging, Thank you!

File: linchpin/ansible_runner.py [hunk 2]
```diff
@@ -82,6 +82,28 @@ def subprocess_runner(cmd, shell=False):
     return proc.returncode, stdout, stderr
 
 
+def ansible_runner_2x(playbook_path,
+                      extra_vars,
+                      options=None,
+                      inventory_src='localhost',
```
Comment (samvarankashyap) [inline linchpin/ansible_runner.py]: @JunqiZhang0  Why is ansible runner changed in this PR?
I think it is reverted back. 
could you drop the commit?

**Comment by JunqiZhang0** [comment]
[merge]

**Comment by JunqiZhang0** [comment]
[test]

**Pull request merged**